src/components/inputs/number/number.component.tsx

```diff
@@ -22,6 +22,11 @@ const NumberField: React.FC<FormFieldInputProps> = ({ field, value, errors, warn
     }
   };
 
+  const handleChange = (event) => {
```

Comment:
Not a big deal but can we wrap this in a `useCallback(..)`?

Reply:
Done